engine/dlib/src/dlib/configfile.h

```diff
@@ -15,7 +15,7 @@
 #ifndef DM_CONFIGFILE_H
 #define DM_CONFIGFILE_H
 
-#include <dmsdk/dlib/configfile.h>
+#include <dmsdk/dlib/configfile_gen.hpp>
```

Comment:
Here's another tricky thing. This configfile.h is a C++ header, and can include our generated hpp header.
This allows us to internally continue to use the C++ api without any changes.

Reply:
"dmsdk headers"

Well, if there is a `configfile.hpp` use that for the C++ sdk, otherwise use `configfile_gen.hpp` if that exists.

Or, use `configfile.h` (the C api).

The manually created c++ header is only created if it was too tricky to create a generated api. For config file, it worked well to generate, so no manually created header was needed.